src/test/java/uk/gov/ida/matchingserviceadapter/builders/AttributeStatementBuilder.java

```diff
@@ -1,153 +1,21 @@
 package uk.gov.ida.matchingserviceadapter.builders;
 
-import org.joda.time.LocalDate;
 import org.opensaml.saml.saml2.core.Attribute;
 import org.opensaml.saml.saml2.core.AttributeStatement;
-import uk.gov.ida.saml.core.IdaConstants;
 import uk.gov.ida.saml.core.OpenSamlXmlObjectFactory;
-import uk.gov.ida.saml.core.extensions.eidas.BirthName;
-import uk.gov.ida.saml.core.extensions.eidas.CurrentFamilyName;
-import uk.gov.ida.saml.core.extensions.eidas.CurrentGivenName;
-import uk.gov.ida.saml.core.extensions.eidas.DateOfBirth;
-import uk.gov.ida.saml.core.extensions.eidas.Gender;
-import uk.gov.ida.saml.core.extensions.eidas.PersonIdentifier;
-import uk.gov.ida.saml.core.extensions.eidas.PlaceOfBirth;
-import uk.gov.ida.saml.core.extensions.eidas.impl.BirthNameBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.CurrentFamilyNameBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.CurrentGivenNameBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.DateOfBirthBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.GenderBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.PersonIdentifierBuilder;
-import uk.gov.ida.saml.core.extensions.eidas.impl.PlaceOfBirthBuilder;
 
 import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
 import java.util.List;
 
 public class AttributeStatementBuilder {
     private static OpenSamlXmlObjectFactory openSamlXmlObjectFactory = new OpenSamlXmlObjectFactory();
     private List<Attribute> attributes = new ArrayList<>();
 
-    public static AttributeStatementBuilder anAttributeStatement(Attribute ... attributes) {
-        return new AttributeStatementBuilder().addAllAttributes(attributes != null ? Arrays.asList(attributes) : Collections.emptyList());
-    }
-
```

Comment:
This doesn't seem right. `attributes` is a private field and since the `addAttribute` and `addAllAttributes` methods below have been removed, there is no way for an external caller to use this param or add attributes. If it is the case that we only create new attribute statements with empty attributes list, then this field should be removed for internal consistency of this class.